Resources/Prototypes/Entities/Structures/Machines/ftldrive.yml

```
@@ -0,0 +1,37 @@
+    - type: Appearance
+    - type: AmbientOnPowered
+    - type: AmbientSound
+      volume: -9
+      range: 5
+      sound:
+        path: /Audio/Ambience/Objects/server_fans.ogg
+    - type: GenericVisualizer
+      visuals:
+        enum.PowerDeviceVisuals.Powered:
+          enum.PowerDeviceVisualLayers.Powered:
+            True: { visible: true }
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)